app/helpers/security_group_helper/textual_summary.rb

```diff
@@ -40,9 +40,9 @@ def textual_group_firewall
       [
         rule.network_protocol,
         rule.host_protocol,
-        rule.direction,
+        _(rule.direction),
```

Comment:
Are the possible values for this already in the catalog?

Reply:
yeah this is outbound and inbound.
![Screen Shot 2022-06-14 at 2 39 26 PM](https://user-images.githubusercontent.com/29209973/173664554-3eb74ecd-688c-42b8-86bd-b9f4bb015aac.png)
![Screen Shot 2022-06-14 at 2 38 51 PM](https://user-images.githubusercontent.com/29209973/173664561-260a012a-dea9-4f8d-9b28-fb59aa09f8b2.png)